libconcurrent/includes/dsmstack.h

```diff
@@ -9,7 +9,7 @@
 
 typedef struct DSMStackStruct {
```

Comment:
General comment: In structs definition you can omit the name in the first line:
`typedef struct DSMStackStruct {` --> `typedef struct {`

Reply:
In some structs, yes it could be done.
However, in case we need a 'next' field there is no other way.